components/admin_console/schema_admin_settings.jsx

```diff
@@ -1039,6 +1047,12 @@ export default class SchemaAdminSettings extends React.Component {
                         onMouseOver={this.openTooltip}
                         onMouseOut={this.closeTooltip}
                     >
+                        <FormError
```

Comment:
What happens here if we have both a `clientWarning` and `serverError` ? I'm not sure if it's a possible scenario but if it does happen, do the text render properly without overlap?

Reply:
yea they render on two separate lines - doesnt look the greatest but it works

Reply:
I see. Do you think it makes sense to provide a screenshot for Michael to review?